### PR TITLE
Move in (+quickfix) CSS for ItemPageTable from SPC

### PR DESCRIPTION
**Pull request by alexkb0009**
2020-08-10 - Pushed SPC PR https://github.com/4dn-dcic/shared-portal-components/pull/36 which removes this block of CSS there

### TODO:
- Merge above SPC PR, release new SPC version (SPCv).
- Update shared-portal-components version (SPCv) in package.json, along w. lockfile in this branch, review+merge.

### CODE REVIEW

File: src/encoded/static/scss/encoded/modules/_search.scss [hunk 1]
```diff
@@ -149,6 +149,83 @@ $search-results-above-results-row-height: 40px;
...
+			
+		}
+
+		.table-row {
+			.column {
+				@include search-result-column-block;
+				float: none;
+				display: inline-block;
+				vertical-align: middle;
+				&:first-child {
```
Comment (alexkb0009) [inline src/encoded/static/scss/encoded/modules/_search.scss:181]: Updated to :first-child from [data-first-visible-column] as latter is not case for ItemPageTable. Was typo made when `[data-first-visible-column]` was first introduced (and copied-pasted into here, when shouldn't have been).

File: src/encoded/static/components/item-pages/FileView.js [hunk 2]
```diff
@@ -92,59 +92,53 @@ export default class FileView extends WorkflowRunTracingView {
...
 
-    static propTypes = {
-        'context' : PropTypes.shape({
-            'experiments' : PropTypes.arrayOf(PropTypes.shape({
-                'experiment_sets' : PropTypes.arrayOf(PropTypes.shape({
-                    '@id' : PropTypes.string.isRequired
-                }))
-            })),
+function FileViewOverview (props) {
+    const { context, windowWidth, width, schemas, href } = props;
```
Comment (j1z0) [inline src/encoded/static/components/item-pages/FileView.js:97]: Codacy found an issue: ['windowWidth' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5937521)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5937521&bid=19461439)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5937521&bid=19461439)

File: src/encoded/static/components/item-pages/FileView.js (hunk 2 above)
Comment (j1z0) [inline src/encoded/static/components/item-pages/FileView.js:97]: Codacy found an issue: ['windowWidth' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5937521)

**Pull request merged**